rs/ic_os/utils/src/lib.rs

```diff
@@ -14,70 +14,45 @@ pub fn get_command_stdout<'a, StringIter: IntoIterator<Item = &'a str>>(
     command: &str,
     args: StringIter,
 ) -> Result<String> {
-    let mut cmd = Command::new(command);
-    let mut arg_string: String = String::new();
-    // TODO - replace loop with single chain call pipeline.
-    for arg in args {
-        cmd.arg(arg);
-        arg_string.push_str(arg);
-        arg_string.push(' ');
-    }
-    let output = cmd.output()?;
+    let args_vec: Vec<&str> = args.into_iter().collect();
+
+    let output = Command::new(command).args(&args_vec).output()?;
```

Comment:
I think, args should be able to take the `args: StringIter` directly?

Reply:
Done, WDYT? I had to clone args and then construct args_vec inside the failure condition

Reply:
We're in the weeds for sure 😅.

I'm not sure how we even got to this crazy type:
```
pub fn get_command_stdout<'a, StringIter: IntoIterator<Item = &'a str> + Clone>(
    command: &str,
    args: StringIter,
) -> Result<String>
```

 Can we just use a slice instead? It looks like things should work out simpler across the board if we instead:
 ```
 pub fn get_command_stdout(command: &str, args: &[&str]) -> Result<String>
 ```

Reply:
Much better! b41f03d88c6280f9809c8ecd2df622fd638e9fea

Reply:
Awesome! 🤩

Last thing, in testing this I noticed that we print `output.stderr` as a `Vec<u8>`. Could we give it the same `String::from_utf8(output.stderr)?` treatment, so that we can print it as actual text (and without debug `{:?}`).

Reply:
Good catch! Done! 8aa4251372b22575f0135d6a368e27ff9f68ddf4